src/buildCommon.js

```diff
@@ -183,6 +184,12 @@ const mathit = function(
             fontName: "Main-Italic",
             fontClass: "mainit",
         };
+    } else if (value.charCodeAt(0) === 0xD835) {
+        const [wideFontName, wideFontClass] = wideCharacterFont(value, mode);
+        return {
+            fontName: wideFontName,
+            fontClass: wideFontClass,
+        };
```

Comment:
I think these logics should be in `mathDefault` or `makeOrd`. Currently `\mathit{𝔄}` produces `𝔄` with metrics for `A`.

Reply:
Well, that's a problem. Thank you for pointing this out. I will fix it, but maybe not right away. I have a busy few days ahead.

Reply:
@ylemkimon I took a look and things are in this PR fine as they are. To test, I did a `console.log(metrics);` on a few letters. A clear example is the letter "z".  

*  In default Main-Italic, the depth of letter"z" should be zero and that is exactly what the console reports. 
* In Fraktur, the depth of letter "𝔷", per `fontMetrics.js`, should be 0.18906. Again, that's exactly what the console reports when I input "𝔷".
* When I input `frak{z}`, the console again reports a depth of 0.18906

Remember that the data in fontMetrics.js` does not go up to an index value of U+1D400. The letters in the KaTeX fonts are placed at their ASCII char codes, not their Unicode code points. The data is indexed the same way. So it's correct to ask for metrics of the letter "z" in my example, or "A" in your example.

I don't know how you got metrics for `\mathit{𝔄}`. When I tried to input `\mathit{𝔄}`, KaTeX gave me a parse error. If it would parse, one could argue that the proper metrics would indeed be for "A". According to @rrandallcainc in PR #1111, math commands, such as `\mathit` "change all aspects of the font (weight, shape, and family)".

I think this PR, as written, is getting the proper metrics.

Reply:
@ronkok I checked out your branch and using `npm start` it renders `\mathit{𝔄}` without `ParseError`:

![image](https://user-images.githubusercontent.com/888148/37755101-6924bd24-2de7-11e8-9594-cb27f17a4ab6.png)

Regarding the metrics, I was looking at the old source code and got wrong, sorry. It uses metrics of 𝔄 and produces 𝔄.

The question is, should math font commands alter them? Currently, `mathit` renders as-is, e.g., `\mathit{𝔄}` gives 𝔄, whereas other font commands change its font, e.g., `\mathrm{𝔄}` gives A.

Reply:
Also, it seems that the `\boldsymbol` doesn't bolden given character, instead it renders in bold italic.

Reply:
Regarding their behavior, I think it should follow #1217, unless LaTeX commands exist to typeset them in the text mode.

Reply:
> Also, it seems that the `\boldsymbol` doesn't bolden given character, instead it renders in bold italic.

A bold italic is exactly what `\boldsymbol` should do. It's part of the AMS package. Here's an rendering of `\boldsymbol{ABD}` from quicklatex.com:  ![](http://quicklatex.com/cache3/dc/ql_d43e2ae8fd21fea1ce156d878291cfdc_l3.svg)

Should `\mathit{𝔄}` override the Fraktur font and render an *A*? That's an interesting question. I'll do some research into how XeTeX behaves. If the answer turns out to be yes, that will take me some time. I'm booked up for the next few days.

Reply:
So `\boldsymbol{𝖠}` should give 𝑨 rather than 𝝖, right?

Reply:
So `\boldsymbol{𝖠}` should give 𝑨 rather than 𝝖, right?

Yep.